paper/paper.md

```diff
@@ -0,0 +1,141 @@
+
+## CDP Architecture
+
+Council Data Project consists of three primary tools:
+
+1. [cookiecutter-cdp-deployment](https://github.com/CouncilDataProject/cookiecutter-cdp-deployment): A Python [cookiecutter](https://cookiecutter.readthedocs.io/) [@cookiecutter] template to assist users in fully deploying a new CDP instance. A "CDP Instance" is a unique deployment of CDP software and tools. For example, there is an "instance" of CDP for the ["Seattle City Council"](https://councildataproject.org/seattle/#/) and an instance of CDP for the "King County Council". Each instance is comprised of it's own repository, database, file storage bucket, processing pipelines, and web application.
+
+2. [cdp-backend](https://github.com/CouncilDataProject/cdp-backend): A Python package containing CDP's database schema definition, a file format for transcripts generated by speech-to-text algorithms, an infrastructure specification, and processing pipelines. This package currently contains an event gather and processing workflow that will parse event details, generate a transcript for the event using either the provided closed caption file, or using Google Speech-to-Text from the provided event video, and finally, generate and store event metadata (voting records, thumbnails, minutes items, etc.) This package additionally provides a workflow for generating a TF-IDF based event index for weighted term search. Processing workflows (and all tools in `cdp-backend`) are separate from any one CDP instance so that all CDP instances can be easily upgraded whenever there is a new version of `cdp-backend` released.
+
+3. [cdp-frontend](https://github.com/CouncilDataProject/cdp-frontend): A TypeScript library containing all frontend components and a compiled web application. The web application allows for simple data exploration and sharing, and as such, acts as a method for providing context for events, voting records, and more because it allows users to interactively explore the data produced by the backend pipelines. The web application (and all components in `cdp-frontend`) is separate from any single CDP instance so that all CDP instances can be easily upgraded whenever there is a new version of `cdp-frontend` released.
+
+## Cookiecutter and the Produced Repository
+
+`cookiecutter-cdp-deployment` will generate all necessary files for an entirely new CDP instance as well as additional setup documentation for the user to follow to fully complete the instance deployment process.
+
+Utilizing [GitHub Actions](https://github.com/features/actions) and [GitHub Pages](https://pages.github.com/), data processing and web hosting are entirely free as long as the user sets their instance's GitHub repository visibility to public. While the limited primary costs of CDP tools are:
```

Comment:
I would move "While the limited primary costs of CDP tools are:" to a newline and adjust it to be:

"Deploying a CDP instance incurs some small primary costs by using:"

(or something along these lines)